e2e/page-objects/onboarding.ts

```diff
@@ -25,7 +25,7 @@ export default class Onboarding {
 	}
 
 	get continueButton() {
-		return this.locator.getByRole( 'button', { name: 'Add site' } );
+		return this.locator.getByRole( 'button', { name: 'Continue' } );
```

Comment:
This plan should work:
1. Adjust this condition to pass on both scenarios
2. Merge this PR
3. Open new PR to update the commit hash in `metrics-comparison.yml` to the trunk commit with this update change and adjust the condition back to `name: Continue`

What do you think?

```suggestion
		return this.locator.getByRole( 'button', { name: /Continue|Add site/ } );
```